sunbird_service_api_test/src/test/java/org/sunbird/integration/test/common/BaseCitrusTestRunner.java

```diff
@@ -168,6 +168,15 @@ public void getAuthToken(
     }
   }
 
+  public void updateUserRequiredLoginActionTest(
```

Comment:
make it private method, and rename getAuthToken to getUserAuthToken which can use this method when isUserAuthrequired set to true